parse/file.go

```diff
@@ -43,6 +43,9 @@ func parseStructTypeSpec(ts *ast.TypeSpec, str *ast.StructType) (*StructInfo, er
 		if len(tag) == 0 {
 			continue
 		}
+		if tag == "-" {
```

Comment:
Is it a kind of code style to write like this:

```
                if len(tag) == 0 {
			continue
		}
		if tag == "-" {
			continue
		}
```

instead of `if len(tag) == 0 || tag == "-"`?